articles/install-vpp-apps-on-macos-using-fleet.md

```diff
@@ -0,0 +1,138 @@
+# Install VPP apps on macOS using Fleet
```

Comment:
```suggestion
# Install App Store apps (VPP) on macOS using Fleet
```

Suggest tweaking the title.

I think most IT admins know what VPP is but I think let's strive to catch the attention of the general audience or someone very new to IT.